app/views/students/show.html.erb

```diff
@@ -1,8 +1,8 @@
 <main style="padding-left: 240px">
-<div class="row">
-  <div class="col s12"><p></p></div>
-  <div class="col s12 m4 l2"><p></p></div>
-  <%= render "student_info", user: @user %>
-  <div class="col s12 m4 l2"><p></p></div>
-</div>
+  <div class="row">
+    <div class="col s12"><p></p></div>
+    <div class="col s12 m4 l2"><p></p></div>
+    <%= render "student_info", user: @user, student: @student %>
```

Comment:
Why are you sending the `user` and `student` parameters and not using them in `_student_info.html.erb`?

Reply:
My bad. Juxt fixed it

Reply:
thanks! :+1: